<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#444444;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="removeSubTreeBut">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Remove Sub Tree</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>542</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #444444;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="insertBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBut">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>470</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="findSpin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="findLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 	background-color:#333333;
	color: white;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="removespin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printPreOrderBut">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Print PreOrder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printInOrderBut">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Print InOrder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printPostOrderBut">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Print PostOrder</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="preOrderEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inOrderEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="postOrderEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="removeSubTreeSpin">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="subTreeRight">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    </string>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="subTreeLeft">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
   </string>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="balanceBut">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Balance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSubTreeBut">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Add Sub Tree</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>89</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#444444;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTreeWidget" name="secondtreeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>421</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #444444;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="removeBut_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="removespin_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="insertBut_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmBut">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
